src/Karma/Application.php

```diff
@@ -119,10 +121,36 @@ private function initializeSourceFileSystem()
             {
                 $adapter->mount($path, new Local($path));
             }
-             
+
+            return $adapter;
+        };
+
+        $this['target.fileSystem.adapter'] = function($c) {
+
+            if($c['target.path'] !== $c['sources.path'])
+            {
+                $c['hydrator.allowNonDistFilesOverwrite'] = true;
+            }
+
+            if(! is_array($c['target.path']))
+            {
+                return new Local($c['target.path']);
+            }
+
+            $adapter = new MultipleAdapter();
+
+            foreach($c['target.path'] as $path)
+            {
+                $adapter->mount($path, new Local($path));
```

Comment:
How do you manage mapping between multiple source paths and multiple target paths ?

What if cardinalities are different ? if names are different ? 
This feature makes sense for unique source path.